cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/draw-shape-popover.tsx

```diff
@@ -71,7 +77,39 @@ function DrawShapePopoverComponent(props: Props): JSX.Element {
                 </Col>
             </Row>
             {
-                shapeType !== ShapeType.RECTANGLE && (
+                shapeType === ShapeType.RECTANGLE ? (
+                    <>
+                        <Row>
+                            <Col>
+                                <Text className='cvat-text-color'> Drawing method </Text>
+                            </Col>
+                        </Row>
+                        <Row type='flex' justify='space-around'>
+                            <Col>
+                                <Radio.Group
+                                    style={{ display: 'flex' }}
+                                    defaultValue={BoxDrawingType.BY_TWO_POINTS}
+                                    onChange={(event: RadioChangeEvent): void => {
```

Comment:
Maybe shouldn't we create one more lambda here?
Why do not just pass ``onChangeBoxDrawingType(event:  RadioChangeEvent): void`` to this component from the container

Reply:
Thought it would be better to keep the parameter as simple data type as in other named call back functions. But directly passing the event seems to be more readable. Fixed [here](https://github.com/opencv/cvat/pull/1127/commits/865c0aa3ec910d4ed441766208329fdbea9561ce#diff-789be1c3834d95b25049c8d995578c19R92).